torch/_functorch/_aot_autograd/jit_compile_runtime_wrappers.py

```diff
@@ -567,7 +568,6 @@ class CompiledFunction(torch.autograd.Function):
         maybe_subclass_metadata: Optional[SubclassMeta] = maybe_subclass_meta
         num_symints_saved_for_bw = _num_symints_saved_for_bw
         _compiled_autograd_should_lift = False
-        _fakify_first_call = fakify_first_call
```

Comment:
This is now handled by FakifiedOutWrapper, so we don't need to track it anymore in CompiledFunction